src/components/bf-navigation/HelpMenu/HelpMenu.vue

```diff
@@ -21,10 +21,19 @@
           <li>
             <a
               class="bf-menu-item"
-              href="#"
-              @click.prevent="showIntercom"
+              href="https://forms.clickup.com/8664796/f/88dpw-2951/I6M7Z1V8NEBQFTIKSU"
+              target="_blank"
+            >
+              Contact Us
+            </a>
+          </li>
+          <li>
+            <a
+              class="bf-menu-item"
+              href="https://docs.pennsieve.io/page/pennsieve-open-office-hours"
```

Comment:
I think the link here should be: https://docs.pennsieve.io/docs/pennsieve-open-office-hours

Reply:
fixed! Thanks so much for catching that typo.